src/sources/ljm/test.py

```
@@ -0,0 +1,135 @@
+    RATE=1,
+    SCAN_RATE=1,
+    SCANS_PER_READ=1,
+    READ_BULK=1,
+    setup=lambda: None,
+)
+sys.modules["config"] = fake_config
+
+# provide a lightweight msgpack stub used by main
+import types as _types
+fake_msgpack = _types.ModuleType("msgpack")
+fake_msgpack.packb = lambda x: b""
```

Ruff (0.15.5)

[warning] Unused lambda argument: `x`

(ARG005)